pool/pool.go

```diff
@@ -1229,7 +1230,7 @@ func (p *Pool) checkSessionTokenErr(err error, address string) bool {
 	return false
 }
 
-func createSessionTokenForDuration(ctx context.Context, c client, ownerID user.ID, dur uint64) (*session.Token, error) {
+func createSessionTokenForDuration(ctx context.Context, c client, ownerID user.ID, dur uint64) (*session.Object, error) {
```

Comment:
`ownerID` is not used in this function. Will we set owner for token somewhere else?

Reply:
Owner is set automatically in `Sign` method.